.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - '[0-9]+.[0-9]+'
+  schedule:
+    - cron:  '0 4 * * *'
+jobs:
+  Test:
+    name: Run tests
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        jdk: [8, 11]
```

```suggestion
        jdk: [8]
```

As for other CMs, we can test with Java 8 only

---

ok